app/components/ConnectionSettings.vue.ts

```diff
@@ -43,7 +44,12 @@ export default class SceneTransitions extends Vue {
   }
 
   get sceneOptions() {
-    return this.scenesService.scenes.map(scene => ({ title: scene.name, value: scene.id }));
+    const sceneOptions = this.scenesService.scenes.map(scene => ({
+      title: scene.name,
+      value: scene.id,
+    }));
+    sceneOptions.unshift({ title: $t('All'), value: 'ALL' });
+    return sceneOptions;
```

Comment:
Super nitpicky, but a cleaner way to write this might be:
```TypeScript
return [
  { title: $t('All'), value: 'ALL' },
  ...this.scenesService.scenes.map(scene => ({
    title: scene.name,
    value: scene.id,
  }))
];
```
The primary advantage is that you're composing the object you want to return, rather that using a temporary variable, mutating it, then returning it.